atom/app/atom_content_client.cc

```diff
@@ -116,9 +116,9 @@ AtomContentClient::AtomContentClient() {
 AtomContentClient::~AtomContentClient() {
 }
 
-void AtomContentClient::SetActiveURL(const GURL& url) {
-  base::debug::SetCrashKeyValue(crash_keys::kActiveURL,
-                                url.possibly_invalid_spec());
```

Comment:
This method isn't called anywhere, do you mean we need to do that?

Reply:
I mean we have to do this
```
base::debug::SetCrashKeyValue(crash_keys::kActiveURL,
                              url.possibly_invalid_spec());
static crash_reporter::CrashKeyString<64> top_origin_key("top-origin");
top_origin_key.Set(top_origin);
```
you can't just discard the line setting crash key for url no matter we are calling it or not

Reply:
`base::debug::SetCrashKeyValue()` is the deprecated API that this commit is updating. The crash key is set with the new code.

Reply:
right, I mean you have to do something for url arg like
```
static crash_reporter::CrashKeyString<1024> active_url("url-chunk");
active_url.Set(url.possibly_invalid_spec());
```
instead of completely delete the lines and not using it